simplified-app-vanilla/src/main/java/org/nypl/simplified/vanilla/VanillaBuildConfigurationService.kt

```diff
@@ -15,6 +15,8 @@ class VanillaBuildConfigurationService : BuildConfigurationServiceType {
     get() = true
   override val showHoldsTab: Boolean
     get() = true
+  override val showBooksFromAllAccounts: Boolean
+    get() = true
```

Comment:
Can we make this false so it works more like SimplyE?

Reply:
The point was to make it _not_ like SimplyE so that you could see the difference. :laughing:

Reply:
Maybe we could make this a debug preference

Reply:
Deja vu!

Reply:
Fine as-is though! Let's just get it merged.